test/langtools/tools/javac/valhalla/lworld-values/CheckSync.java

```diff
@@ -21,7 +21,6 @@ void foo() {
             notify();
             notifyAll();
             finalize();
-            clone();
         }
```

Comment:
Rather than deleting this line, we could have simply commented saying clone is not an error anymore - that would have minimized the changes to the expected out file (we call them the golden files). No need to change now, but in future you can use such techniques to minimize diffs that are painful/tedious to verify.

Reply:
Thanks you for the recommendation!